srcpkgs/spectrwm/template

```diff
@@ -1,26 +1,32 @@
 # Template file for 'spectrwm'
 pkgname=spectrwm
-version=3.5.1
+version=3.6.0
 revision=1
 _distname=SPECTRWM_${version//./_}
 build_style=gnu-makefile
 make_build_args='-C linux SYSCONFDIR=/etc'
 make_install_args="$make_build_args"
 hostmakedepends="pkg-config"
 makedepends="libXcursor-devel libXft-devel libXrandr-devel libXt-devel
- xcb-util-keysyms-devel xcb-util-wm-devel"
+ xcb-util-keysyms-devel xcb-util-wm-devel libbsd-devel"
 short_desc="Small dynamic tiling window manager for X11"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="ISC"
 homepage="http://spectrwm.org"
 distfiles="https://github.com/conformal/spectrwm/archive/${_distname}.tar.gz"
-checksum=7fe4260208a52d9fb844291d0afe6af18887c5452df0b921ed780af7b9223f0f
+checksum=49854c5215475d2450d2558da12a3afe9019861170ba9d556c8b403d4d66f0d1
 conf_files="/etc/spectrwm.conf"
 
 if [ "$XBPS_TARGET_LIBC" = "musl" ]; then
 	makedepends+=" musl-legacy-compat"
+else
+	makedepends+=" libbsd-devel"
 fi
 
 post_install() {
 	vlicense LICENSE.md
+	vdoc spectrwm.html
+	vinstall spectrwm.desktop 644 usr/share/xsessions
+	vinstall baraction.sh 755 usr/share/spectrwm
+	vinstall screenshot.sh 755 usr/share/spectrwm
```

Comment:
```suggestion
```
spectrwm.html is just an html version of the manpage already in the package.

the makefile already installs the other files. this is redundant